### ISSUE TITLE
ActionCable -> RemoteConnection class - keyword deprecation warning

### ISSUE THREAD

**Issue by thomasbalsloev** [rails/rails#41699]
I get a deprecation when a cable connection is disconnected.
I'm disconnecting the cable on user logout by running this line in the session controller:

`ActionCable.server.disconnect(current_verified_user: current_user, session_id: session.id)`

I can reproduce by doing this in the rails console:
```
2.7.0 :001 > ActionCable.server.disconnect(current_verified_user: {}, session_id: 'abcabc')
/usr/local/rvm/rubies/ruby-2.7.1/lib/ruby/gems/2.7.0/gems/actioncable-6.0.3.5/lib/action_cable/remote_connections.rb:48: warning: Passing the keyword argument as the last hash parameter is deprecated
/usr/local/rvm/rubies/ruby-2.7.1/lib/ruby/gems/2.7.0/gems/actioncable-6.0.3.5/lib/action_cable/server/broadcasting.rb:24: warning: The called method `broadcast' is defined here
[ActionCable] Broadcasting to action_cable/abcabc:{}: {:type=>"disconnect"}
 => nil 
```

I have tracked the offending code to this line:

https://github.com/rails/rails/blob/5e300f1121dd24204cf0586521f34632485fce15/actioncable/lib/action_cable/remote_connections.rb#L48

To fix this warning I change the last parameter to a hash: server.broadcast internal_channel, { type: "disconnect" }

### Expected behavior
No deprecation warning.

### Actual behavior
Deprecation warning appears:
```
/usr/local/rvm/rubies/ruby-2.7.1/lib/ruby/gems/2.7.0/gems/actioncable-6.0.3.5/lib/action_cable/remote_connections.rb:49: warning: Passing the keyword argument as the last hash parameter is deprecated
/usr/local/rvm/rubies/ruby-2.7.1/lib/ruby/gems/2.7.0/gems/actioncable-6.0.3.5/lib/action_cable/server/broadcasting.rb:24: warning: The called method `broadcast' is defined here
```

Stacktrace:
```
/usr/local/rvm/rubies/ruby-2.7.1/lib/ruby/gems/2.7.0/gems/actioncable-6.0.3.5/lib/action_cable/remote_connections.rb:48:in `disconnect'
/usr/local/rvm/rubies/ruby-2.7.1/lib/ruby/gems/2.7.0/gems/actioncable-6.0.3.5/lib/action_cable/server/base.rb:38:in `disconnect'
/var/www/portal/app/controllers/auth/sessions_controller.rb:24:in `disconnect_action_cable'
```
### System configuration
**Rails version**: 6.0.3.5
**Ruby version**: ruby 2.7.1p83 (2020-03-31 revision a0c7c23c9c) [x86_64-linux]

**Comment by zzak** [comment]
I think this was fixed in d2642762885d79b330930ea473357f965a0b5581 can you check it?

**Comment by kamipo** [comment]
Fixed by d2642762885d79b330930ea473357f965a0b5581.
